openedx/core/lib/exceptions.py

```diff
@@ -0,0 +1,22 @@
+""" Common Purpose Errors"""
+from django.core.exceptions import ObjectDoesNotExist
+
+
+class CourseNotFoundError(ObjectDoesNotExist):
+    """ Course was not found. """
+    pass
+
+
+class ThreadNotFoundError(ObjectDoesNotExist):
+    """ Thread was not found. """
+    pass
+
+
+class CommentNotFoundError(ObjectDoesNotExist):
```

Comment:
`CommentNotFoundError` and `ThreadNotFoundError` are specific to the discussion forums project and shouldn't be part of the openedx core project, right?